velox/connectors/Connector.h

```diff
@@ -259,7 +260,8 @@ class ConnectorQueryCtx {
       const std::string& queryId,
       const std::string& taskId,
       const std::string& planNodeId,
-      int driverId)
+      int driverId,
+      folly::CancellationToken cancellationToken)
```

Comment:
should we have an empty default value for this so that we don't need to change all callsites from this function?

Reply:
Yes, it is documented.